shared/common-adapters/web-view.desktop.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from 'react'
+import type {WebViewProps} from './web-view'
+
+class WebView extends React.PureComponent<WebViewProps> {
+  _webviewRef: ?WebviewElement
+
+  _setWebviewRef = (r: ?HTMLElement) => {
+    this._webviewRef = (r: any)
```

Comment:
@akalin-keybase so I poked around and this is the best I could type it. Please let me know if you have better ideas!

Reply:
This can be
```
_setWebviewRef = (r: ?WebviewElement) => {
```
if you're in this area of the code again!

Reply:
I think I tried it, but flow didn't like me passing a `(r: ?WebviewElement): void` to `ref`. It expects ` `(r: ?HTMLElement): void`.